samples/openapi3/client/petstore/python-experimental/petstore_api/model/nullable_shape.py

```diff
@@ -94,14 +95,14 @@ def _composed_schemas(cls):
         # code would be run when this module is imported, and these composed
         # classes don't exist yet because their module has not finished
         # loading
-        oneOf_2 = NoneSchema
+        oneOf_2 = AnyTypeSchema
```

Comment:
Something is not working right here. The schema is:
```
    NullableShape:
      description: The value may be a shape or the 'null' value.
        The 'nullable' attribute was introduced in OAS schema >= 3.0
        and has been deprecated in OAS schema >= 3.1.
        For a nullable composed schema to work, one of its chosen oneOf schemas must be type null
      oneOf:
        - $ref: '#/components/schemas/Triangle'
        - $ref: '#/components/schemas/Quadrilateral'
        - type: null
      discriminator:
        propertyName: shapeType
      nullable: true
```
So the 3rd oneOf schema should be NoneSchema NOT AnyTypeSchema.

Reply:
Yup, could be a regression. I tried updating the parser to 2.0.30 (released several hours ago) but no luck (no change in samples).

Can you please report the issue to swagger parser directly with the details when you've time?

Reply:
Yup, it could also be our code not seeing the null type right. If it is the parser I will report it.

Reply:
I did some investigating here:
1. For yaml `type: null` the schema is defined as an type: null AnyType schema by swagger-parser
2. For yaml `type: "null"` the schema is correctly defined as type: "null" NoneSchema by swagger-parser
So I will:
- Update the spec to make the type be the quoted null value
- Remove the discriminator from that schema because per the openapi spec, when discriminator exists, then the discriminator key MUST exist in the payload. a null payload value cannot meet that constraint.